src/pages/ReportParticipantsPage.js

```diff
@@ -73,39 +73,43 @@ const ReportParticipantsPage = (props) => {
     const participants = getAllParticipants(props.report, props.personalDetails);
 
     return (
-        <ScreenWrapper>
+        <ScreenWrapper includeSafeAreaPaddingBottom={false}>
             <FullPageNotFoundView shouldShow={_.isEmpty(props.report)}>
-                <HeaderWithCloseButton
-                    title={props.translate((ReportUtils.isChatRoom(props.report) || ReportUtils.isPolicyExpenseChat(props.report)) ? 'common.members' : 'common.details')}
-                    onCloseButtonPress={Navigation.dismissModal}
-                    onBackButtonPress={Navigation.goBack}
-                    shouldShowBackButton={ReportUtils.isChatRoom(props.report) || ReportUtils.isPolicyExpenseChat(props.report)}
-                />
-                <View
-                    pointerEvents="box-none"
-                    style={[
-                        styles.containerWithSpaceBetween,
-                    ]}
-                >
-                    {Boolean(participants.length)
-                        && (
-                        <OptionsList
-                            sections={[{
-                                title: '', data: participants, shouldShow: true, indexOffset: 0,
-                            }]}
-                            onSelectRow={(option) => {
-                                Navigation.navigate(ROUTES.getReportParticipantRoute(
-                                    props.route.params.reportID, option.login,
-                                ));
-                            }}
-                            hideSectionHeaders
-                            showTitleTooltip
-                            disableFocusOptions
-                            boldStyle
-                            optionHoveredStyle={styles.hoveredComponentBG}
+                {({safeAreaPaddingBottomStyle}) => (
```

Comment:
Is this supposed to be a child of the `ScreenWrapper` element or the `FullPageNotFoundView` element? I thought the `ScreenWrapper` element, but here it is nested within the `FullPageNotFoundView` element. Will that break things? It would be best to move this so it is a child of `ScreenWrapper`, no? (Like what you've done in IOUDetailsModal.js).

Reply:
`FullPageNotFoundView` is also nested within `ScreenWrapper`.
This was added intentionally in #14744

Reply:
ah good catch @joelbettner

Reply:
fixed